grafana-plugin/src/models/filters/filters.types.ts

```diff
@@ -1,9 +1,20 @@
+import React from 'react';
+
 import { SelectOption } from 'state/types';
 
 export interface FiltersValues {
   [key: string]: any;
 }
 
+export interface FiltersExtraInformation {
+  [key: string]: {
+    isClearable?: boolean;
+    value?: any;
```

Comment:
isn't it always string?

Reply:
Nope, can be `array`, can be `string`, can be other values